arviz/plots/backends/matplotlib/pairplot.py

```diff
@@ -100,20 +103,34 @@ def plot_pair(
         marginal_kwargs.setdefault("plot_kwargs", {})
         marginal_kwargs["plot_kwargs"].setdefault("linewidth", linewidth)
 
+        point_estimate_marker_kwargs.setdefault("s", figsize[0] + 50)
```

Comment:
Can't this line be moved up, to line 96, instead of being repeated in the if / else block?

Reply:
Thank you for your comments :), in this line I am setting a default value for the marker size as a factor of figure size, which is defined inside the if/else block.

Reply:
Ooh you're right, didn't see that 🤦 All good then!

Reply:
Is there a particular reason why the `markersize` returned by `_scale_fig_size` is not used?